include/xtensor/xfixed.hpp

```diff
@@ -99,7 +99,8 @@ namespace xt
         template <std::size_t I, std::size_t Y, std::size_t... X>
         struct calculate_stride<layout_type::column_major, I, Y, X...>
         {
-            constexpr static std::ptrdiff_t value = Y * calculate_stride<layout_type::column_major, I - 1, X...>::value;
+            constexpr static std::ptrdiff_t value = Y
+                                                    * calculate_stride<layout_type::column_major, I - 1, X...>::value;
```

Comment:
I find this formatting very strange. I think that 
```suggestion
            constexpr static std::ptrdiff_t value = 
                Y * calculate_stride<layout_type::column_major, I - 1, X...>::value;
```
would be much more logical